pom.xml

```diff
@@ -1679,7 +1679,7 @@
             <dependency>
                 <groupId>org.yaml</groupId>
                 <artifactId>snakeyaml</artifactId>
-                <version>1.28</version>
+                <version>1.30</version>
```

Comment:
Why?

Reply:
This was recently updated in #12091 but it looks like the latest jackson version requires this update: https://github.com/airlift/airbase/pull/314 and https://github.com/FasterXML/jackson-dataformats-text/commit/192f452efe62b9ec0d31cfd11ab1bab04b11eff4